tools/vscode_workspace/engine-workspace.yaml

```diff
@@ -0,0 +1,256 @@
+    streambuf: cpp
+    string: cpp
+    string_view: cpp
+    strstream: cpp
+    system_error: cpp
+    tuple: cpp
+    type_traits: cpp
+    typeinfo: cpp
+    unordered_map: cpp
+    unordered_set: cpp
+    valarray: cpp
+    variant: cpp
+    vector: cpp
+    algorithm: cpp
+    filesystem: cpp
+    memory_resource: cpp
+    bit: cpp
+    charconv: cpp
+    format: cpp
+    functional: cpp
+    iterator: cpp
+    utility: cpp
+    __assert: cpp
+    '*.inc': cpp
+    __verbose_abort: cpp
+    '*.def': cpp
+    '*.hpp11': cpp
+    __functional_base: cpp
+    shared_mutex: cpp
+    coroutine: cpp
+    hash_map: cpp
+    hash_set: cpp
+    thread: cpp
+    propagate_const: cpp
+    '*.gen': cpp
+    simd: cpp
+    barrier: cpp
+    cuchar: cpp
+    latch: cpp
+    numbers: cpp
+    scoped_allocator: cpp
+    semaphore: cpp
+    typeindex: cpp
+    __std_stream: cpp
+    '*.ipp': cpp
+    csetjmp: cpp
+    cfenv: cpp
+  C_Cpp.default.includePath:
+    - ${default}
+    - ${workspaceFolder}/..
+    - ${workspaceFolder}
+  dotnet.defaultSolution: disable
+  dart.showTodos: false
+  testMate.cpp.test.advancedExecutables:
+    - name: impeller_unittests_arm64
+      pattern: ../out/host_debug_unopt_arm64/impeller_unittests
+      runTask:
+        before:
+          - impeller_unittests_arm64
+      gtest:
+        prependTestRunningArgs:
+          - --enable_playground
+    - name: display_list_unittests_arm64
+      pattern: ../out/host_debug_unopt_arm64/display_list_unittests
+      runTask:
+        before:
+          - display_list_unittests_arm64
+    - name: impeller_golden_tests_arm64
+      pattern: ../out/host_debug_unopt_arm64/impeller_golden_tests
+      runTask:
+        before:
+          - impeller_golden_tests_arm64
+      gtest:
+        prependTestRunningArgs:
+          - &golden-workspace --working_dir=~/Desktop
+        prependTestListingArgs:
+          - *golden-workspace
+  testMate.cpp.debug.configTemplate:
+    type: cppvsdbg
+    linux:
+      type: cppdbg
+      MIMode: gdb
+    darwin:
+      type: cppdbg
+      MIMode: lldb
+      setupCommands:
+        - description: Source map
+          text: settings set target.source-map "flutter/" "${workspaceFolder}"
+          ignoreFailures: false
+    win32:
+      type: cppvsdbg
+    program: ${exec}
+    args: ${argsArray}
+    cwd: ${cwd}
+    env: ${envObj}
+    environment: ${envObjArray}
+    sourceFileMap: ${sourceFileMapObj}
+tasks:
+  version: 2.0.0
+  tasks:
+    - &et-task
+      label: impeller_unittests_arm64
+      type: shell
+      command: ./flutter/bin/et
+      args:
+        - build
+        - -c
+        - host_debug_unopt_arm64
+        - //flutter/impeller:impeller_unittests
+      options:
+        cwd: ${workspaceFolder}/..
+      problemMatcher:
+        - $gcc
+      presentation:
+        echo: true
+        reveal: always
+        focus: false
+        panel: shared
+        clear: true
+      group:
+        kind: build
+    - <<: *et-task
```

Comment:
Here's one place where the magic is happening.

Reply:
Just checking; `workspaceFolder` didn't seem to work for my regular .vscode configs; does it work in the engine-workspace?

Reply:
It won't work unless you open up a workspace (`//engine.code-workspace`).